Automattic/wp-calypso
# Launchpad.- Update mayberedirect launchpad option retrieval

Related to https://github.com/Automattic/wp-calypso/issues/73830

## Time Estimate

Review: Short
Test: Short

## Proposed Changes

* Add useLaunchpad hook to maybeRedirect controller function. This hook will use the launchpad endpoint to fetch the latest launchpad to determine if the user needs to be redirected to the launchpad screen or `/home`

## Testing Instructions



* Checkout this branch
* Create or use an existing launchpad enabled site (Test on an atomic site as well)
* Navigate to `/home/{siteSlug}` and verify that you are redirected to the launchpad screen
* Navigate to `/home/{siteSlug}?launchpadComplete=true` and verify you are not redirected to the launchpad screen
* Disable the launchpad (change site option `launchpad_screen` to `off` or FALSE) or use a site that is not launchpad enable and navigate to `/home`. Verify that you are not navigated to the launchpad screen.

## Pre-merge Checklist
